tests/test_utils.py

```diff
@@ -265,12 +265,25 @@ def test_force_text(value, expected_text):
         (["-f", "συνδέσεις"], "pip-compile --find-links='συνδέσεις'"),
         (["-o", "my file.txt"], "pip-compile --output-file='my file.txt'"),
         (["-o", "απαιτήσεις.txt"], "pip-compile --output-file='απαιτήσεις.txt'"),
+        # Check ' -- ' (filename escapes)
+        (["--", "--requirements.in"], "pip-compile -- --requirements.in"),
+        # Check '--pip-args' (forwarded) arguments
+        (
+            ["--pip-args", "--disable-pip-version-check"],
+            "pip-compile --pip-args='--disable-pip-version-check'",
+        ),
+        (
+            ["--pip-args", "--disable-pip-version-check --isolated"],
+            "pip-compile --pip-args='--disable-pip-version-check --isolated'",
+        ),
     ],
 )
 def test_get_compile_command(tmpdir_cwd, cli_args, expected_command):
     """
     Test general scenarios for the get_compile_command function.
     """
+    with open("--requirements.in", "w"):
+        pass
```

Comment:
This block could be moved out to a separate test.

Reply:
Sure, next commit will grant that its own test.